src/App.js

```diff
@@ -57,11 +58,26 @@ class App extends Component {
     };
   }
 
+  focusOnAppElement = () => this.appElement.focus();
+
+  genNewSentence = buttonEl => {
+    buttonEl.preventDefault();
```

Comment:
What made you go the prevent default route instead of buttonEl.blur()?
Could using preventDefault() have unintended consequences down the line?
I don't know what the default behavior is besides to focus on the element, but I thought I remember there being more behaviors disabled than just that one.